<?xml version="1.0"?>
<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
<hardware>
  <plugin>shelfbot/FourWheelDriveHardwareInterface</plugin>
  <param name="communication_type">rest</param>
  <param name="base_url">http://shelfbot.local/</param>
</hardware>
</robot>
